### DISCUSSION TITLE
What does publish-osgi-ee:profiles actually do?

### DISCUSSION THREAD

**Discussion by Bananeweizen** [eclipse-tycho/tycho#1667] [Q&A] [answered]
What are the consequences of changing the profiles argument in https://tycho.eclipseprojects.io/doc/latest/tycho-p2/tycho-p2-publisher-plugin/publish-osgi-ee-mojo.html#profiles? E.g. I don't like that the list of known Java EEs is different locally than on a CI system, therefore I want some fixed values there.
So if I choose only 11 there, does that somehow limit the usability of the update site for people running on 17 (assuming my features in the update site are compatible with 11)? I basically don't get what those profiles enable later for end users of the update site.

**Comment by laeubi** [comment, marked as answer]
These profiles are mostly for a ~Bug~ Feature in PDE that requires this to resolve items provided by the JRE... with modular VMs this mostly has become obsolete anyways but no one cared enough to ~fix~ enhance PDE until now.
e.g. Tycho currently fully generates the profiles from the running VM and possibly toolchains that are available to the build.
